README.md

```diff
@@ -0,0 +1,118 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Portfolio Setup <a name="about-project"></a>
+
+Web-Portfolio is an activity created on the first week to show the knowledge we have grasped
+
```

Comment:
- [OPTIONAL] Although the implementation of the project is that of Hello-Microverse, the title of your project is called Web Portfolio. I would advise you in the future to properly and semantically name your repository and README title based on the implementation of the project. For example, you could have named this Hello-Microverse because anyone who sees Web Portfolio will think this repository is your portfolio implementation that contains projects that you want to showcase. 👍 😄